lib/govuk_publishing_components/presenters/public_layout_helper.rb

```diff
@@ -46,17 +46,48 @@ class PublicLayoutHelper
       attr_reader :footer_navigation, :footer_meta, :cookie_banner_data
 
       def initialize(local_assigns)
+        @footer_links_count = generate_ga4_footer_links_count(local_assigns)
         @footer_navigation = local_assigns[:footer_navigation] || navigation_link_generation_from_locale(I18n.t("components.layout_footer.navigation_links"))
```

Comment:
This is where the `Topics / Government Activity` links are generated. Although there is a translation being done here, it looks like there's only an English locale file in the repo, so I haven't had to do any work to ensure things are tracked in English, which would've been a faf!